src/Game/debugInput.cpp

```diff
@@ -0,0 +1,4 @@
+
+#include "debugInput.h"
```

Comment:
Include paths should be "project-absolute", i.e. relative to `src/`. If the .h is in `src/KingSystem/ksysDebugInput.h` then the include should be `#include "KingSystem/ksysDebugInput.h"`.